src/libasr/codegen/asr_to_llvm.cpp

```diff
@@ -1032,9 +1082,14 @@ class ASRToLLVMVisitor : public ASR::BaseVisitor<ASRToLLVMVisitor>
         return free_fn;
     }
 
-    inline void call_lfortran_free_string(llvm::Function* fn) {
-        std::vector<llvm::Value*> args = {tmp};
-        builder->CreateCall(fn, args);
+    inline void call_lcompilers_free_strings() {
+        // if (strings_to_be_deallocated.n > 0) {
```

Comment:
I think we need to uncomment this?

Reply:
Actually this is an incorrect fix. Correct fix should work with both LFortran and LPython.

Reply:
Okay, let's merge it as is, and then I will submit a new PR fixing them.